cmd/server/assets/codestatus/show.html

```diff
@@ -42,6 +42,12 @@ <h5 class="mb-1">Status</h5>
         <h5 class="mb-1">Expiry</h5>
         <span id="code-expires-at" class="sm text-danger"></span>
       </div>
+      {{if .code.HasLongExpires}}
+      <div class="list-group-item">
+        <h5 class="mb-1">Long code expiry</h5>
```

Comment:
I don't think we call this "long code" anywhere but the database. Should we say "SMS verification link" instead?

Reply:
changed to "SMS link expiry"